lib/assets/tmpl-spec.jst

```diff
@@ -57,7 +64,11 @@ describe('${ describe }', function() {
                 filename += '.' + lang;
             }
 
-            filename += '.html';
+            if (engine.name === 'bemhtml') {
```

Comment:
ok, I'm just using the code style from this module, but I prefer the short structure